gerrit_to_github_issues/github_issues.py

```diff
@@ -61,14 +61,13 @@ def remove_duplicated_issue_numbers(issue_dict: dict) -> dict:
     return issue_dict
 
 
-def get_repo(repo_name: str, github_user: str, github_pw: str, github_token: str) -> (github.Github, Repository):
+def get_client(github_user: str, github_pw: str, github_token: str) -> github.Github
```

Comment:
This says it should return a Github client, but it doesn't seem to return anything.

Reply:
Also missing a colon